querybuilder/type_convert.go

```diff
@@ -13,98 +13,94 @@ const (
 )
 
 // String
-func toString(v interface{}) string {
-	switch v.(type) {
+func toString(v interface{}) (interface{}, error) {
```

Comment:
This whole purpose of this package is to give the data in the right type.. Let the signature return the type we expect.
```suggestion
func toString(v interface{}) (string, error) {
```